tests/conftest.py

```diff
@@ -452,17 +483,10 @@ def virtualenv_template(
     # Install setuptools, wheel and pip.
     install_pth_link(venv, "setuptools", setuptools_install)
     install_pth_link(venv, "wheel", wheel_install)
-    pip_editable = tmpdir_factory.mktemp("pip") / "pip"
-    shutil.copytree(pip_src, pip_editable, symlinks=True)
-    # noxfile.py is Python 3 only
-    assert compileall.compile_dir(
-        str(pip_editable),
-        quiet=1,
-        rx=re.compile("noxfile.py$"),
-    )
-    subprocess.check_call(
-        [os.fspath(venv.bin / "python"), "setup.py", "-q", "develop"], cwd=pip_editable
-    )
+
+    pth, dist_info = pip_editable_parts
+    shutil.copy(pth, venv.site)
+    shutil.copytree(dist_info, venv.site, dirs_exist_ok=True, symlinks=True)
```

Comment:
You could try `venv.site.join_path("easy_install.pth").touch()` here.
I suspect the tests rely on the fact tha easy_install.pth pre-exists (and it was created by setup.py develop before this PR). Not sure at all, just a debugging hypothesis.